S3/S3.py

```diff
@@ -1064,6 +1065,27 @@ def set_acl(self, uri, acl):
         response = self.send_request(request)
         return response
 
+    def set_versioning(self, uri, status):
```

Comment:
Instead of taking the status as string directly here, it is probably better to only take an "on/off" value.